script.js

```diff
@@ -1,12 +1,164 @@
-//You can edit ALL of the code here
 function setup() {
+  // this function loads the game of thrones episodes
   const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
 }
 
+// get all shows from shows.js
+const allShows = getAllShows();
+// Sorting the array of objects by shows' name
+allShows.sort(function (a, b) {
+  let nameA = a.name.toLowerCase();
```

Comment:
perfect. you can also use ``localeCompare`` method to achieve this and you don't need to use ``toLowerCase``

Reply:
I didn't know about it. But thanks